CONTRIBUTING.md

```diff
@@ -2,24 +2,23 @@
 # Reporting issues, requesting features
 Please use the one of the forms [here](../../issues/new/choose).
 
-**Please do not report security vulnerabilities through public GitHub issues.** Instead, please report them via the [Autodesk Trust Center](https://www.autodesk.com/trust/contact-us). 
+**Please do not report security vulnerabilities through public GitHub issues.** Instead, please report them via the [Autodesk Trust Center](https://www.autodesk.com/trust/contact-us).
 
 # Contributing code
 We welcome your PRs! We use a fairly standard process:
-1. You sign the Contributor License Agreement.
-2. You submit a PR.
-3. You respond the reviewer's comments.
-4. We merge your PR.
+1. You submit a PR.
+2. Our CLA bot adds a comment in your PR with links to our Contributor License Agreement forms.
+3. You sign the Contributor License Agreement.
+4. You respond the reviewer's comments.
+5. We merge your PR.
 
 ## Contributor License Agreement
 Before contributing code to this project, we ask that you sign a Contributor License Agreement (CLA).
 
-+ [Corporate Contributor License Agreement ("Agreement") v1.0](Autodesk_Form_Corp_Contrib_Agmt_for_Open_Source.pdf) - Please sign this one for corporate use
-+ [Individual Contributor License Agreement ("Agreement") v1.0](Autodesk_Form_Ind_Contrib_Agmt_for_OpenSource.pdf) - Please sign this one if you're an individual contributor
+Once your PR is created, a CLA bot will automatically add a comment with links to our CLA documents, and explain how to sign CLA.
 
-The documents include instructions on where to send the completed forms to.  Once a signed form has been received you will be able to submit pull requests.
 ## Creating PRs and responding to comments
 Please fork the repo and create your PR from the fork. Please respond to the reviewer's comments.
 ## Merging
-Once reviewers are satisfied we will merge your PR. Your changes will appear in the official 
+Once reviewers are satisfied we will merge your PR. Your changes will appear in the official
```

Comment:
This sentence seems to be cut off. I think it should say:
```
Once reviewers are satisfied we will merge your PR. Your changes will appear in the official package.
```

Reply:
Good catch! updated.